src/deluge/dsp/fft/fft_config_manager.cpp

```diff
@@ -27,7 +27,7 @@ ne10_fft_r2c_cfg_int32_t getConfig(int32_t magnitude) {
 	if (magnitude > FFT_CONFIG_MAX_MAGNITUDE)
 		return NULL;
 
-	if (!configs[magnitude]) {
+	if (configs[magnitude] == nullptr) {
```

Comment:
is nullptr correct here?

Reply:
yep, it's typedefed from a ne10_fft_r2c_state_int32_t*